### ISSUE TITLE
[Master Feature] Doc-level infinite scroll

### ISSUE THREAD

**Issue by ericlindley-g** [ampproject/amphtml#14059]
Support document-level infinite scroll, such that when the user reaches the end of the core content of an AMP document (an article, a product detail page, etc), a call can be made to an arbitrary endpoint to fetch additional documents to be dynamically loaded into the view. For the best user experience, users should never see a loading spinner, but instead should either be presented with already-loaded results, or a "recommended content" section to navigate to more.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @ericlindley-g Please add an appropriate category.

**Comment by amedina** [comment]
Is this capability going to be implemented by removing the limitations of `amp-next-page`?

**Comment by ericlindley-g** [comment]
Yep — the implementation of this feature is amp-next-page. It's available to use as an experiment now, and when we've fixed some bugs that have been identified (analytics in particular, as well as signaling the "active" document so the display & sharing URLs can be updated to match), then the feature will launch.

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @aghassemi Do you have any updates?

**Comment by Pkmmte** [comment]
Any updated timeline on this?

**Comment by aghassemi** [comment]
@Pkmmte this feature is currently experimentally available under a `doc-level optin`, please see documentation on how to optin: https://www.ampproject.org/docs/reference/components/amp-next-page

**Comment by wassgha** [comment]
Closing in favor of #25500
